README.md

```diff
@@ -1,9 +1,63 @@
-# Jarvis Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Usman Ullah Asif . Jarvis Consulting
 
+Throughout my education (bachelor’s in electrical engineering and master’s in engineering) I have developed a true passion for code. I believe that data engineering is the most relevant area within IT in the 21st century. As Junior Data Engineer at jarvis and during my deployment in leading Canadian Bank. I was able to particiapte various challenging and fun projects using Agile methodology. Which made me proficent in technologies like java,Scala,Git,Bash,SQL,docker and many more. More than that it taught me about problem solving and Leadership in this Pandemic World.
```

Comment:
Please improve your introduction.. Please use Grammarly to check your grammar and spelling.

Here are some sample profiles
https://github.com/jarviscanada/jarvis_data_eng_BrandonDicicco
https://github.com/jarviscanada/jarvis_data_eng_JohnCao
https://github.com/jarviscanada/jarvis_data_eng_TomasRotbauer
https://github.com/jarviscanada/jarvis_data_eng_TinaLee
https://github.com/jarviscanada/jarvis_data_eng_NarimanAlimuradov
https://github.com/jarviscanada/jarvis_data_eng_JasmeetBrar
https://github.com/jarviscanada/jarvis_data_eng_AlexKirilenko
https://github.com/jarviscanada/jarvis_data_eng_FatimaAnwer